amundsen_application/static/js/components/SearchPage/SearchPanel/index.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { ResourceType } from 'interfaces/Resources';
+import { GlobalState } from 'ducks/rootReducer';
+import { DashboardSearchResults, TableSearchResults, UserSearchResults } from 'ducks/search/types';
+import { TABLE_RESOURCE_TITLE, USER_RESOURCE_TITLE } from 'components/SearchPage/constants';
+import AppConfig from 'config/config';
+
+import './styles.scss';
+
+export interface StateFromProps {
+  selectedTab: ResourceType,
+  tables: TableSearchResults;
+  dashboards: DashboardSearchResults;
+  users: UserSearchResults;
+}
+
+export interface OwnProps {
+  onTabChange: (tab: ResourceType) => void;
+}
+
+export type SearchPanelProps = StateFromProps & OwnProps;
+
+export class SearchPanel extends React.Component<SearchPanelProps> {
+
+  constructor(props) {
+    super(props);
+  }
+
+  onChange = (event) => {
+    this.props.onTabChange(event.target.value);
+  };
+
+  renderRadioOption = (value: ResourceType, label: string, count: number) => {
```

Comment:
Will this eventually be part of a separate `ResourceRadioThing` component?

Reply:
That's the plan. We'll have a generic RadioButton (and maybe a RadioGroup) component where you can just pass in a list of options and an onChange function so we can handle any list of radio options. Ditto on checkboxes too. It might look something like

```
const radioOptions = [{
  value: ResourceType.table,
  label: "Datasets",
  subtitle: "323",
}, { ... }]

return <RadioGroup options={radioOptions} onChange={this.props.selectTab} ...>
```